scripts/utils/batch_request.js

```diff
@@ -0,0 +1,32 @@
+// https://github.com/ethereum/web3.js/issues/1446
+
+module.exports = function (web3) {
+  class BatchRequest {
+    constructor() {
+      this.promises = []
+      this.batch = new web3.BatchRequest()
```

Comment:
Why did you choose this Batching Method over the discussed method on solidity level? Because it is simpler?


Yes, we knew that it won't fix the issue. The hope was just that it would occur less frequently as we are making less requests. If you have a better idea than batching, let us know.